kobuki/2f647af/2f647af.bug

```diff
@@ -151,4 +151,4 @@ fault-codes:
   - PROGRAMMING:CALCULATIONS
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:MODEL-REALITY-DIVERGENCE
+  - SYSTEM:MONITORING
```

Comment:
This bug also seems like a clear case of BEHAVING-INCORRECTLY to me due to incorrect odometry information.

Reply:
My rationale was that messages like odometry and joint states would be used to calculate the inner state of the robot (eventually they might be used to calculate control commands and lead to incorrect behaviour - but we considered such cases as extrapolation before)

Reply:
From the meeting: let's go with both BEHAVING-INCORRECTLY and MONITORING.